src/test/java/nextstep/subway/unit/LineServiceTest.java

```diff
@@ -19,12 +31,19 @@ public class LineServiceTest {
     @Test
     void addSection() {
         // given
-        // stationRepository와 lineRepository를 활용하여 초기값 셋팅
+        Station 첫번째역 = new Station(1L, "첫번째역");
+        Station 두번째역 = new Station(2L, "두번째역");
+        Station 세번째역 = new Station(3L, "세번째역");
+        stationRepository.save(첫번째역);
+        stationRepository.save(두번째역);
+        stationRepository.save(세번째역);
+        LineResponse lineResponse = lineService.saveLine(new LineRequest("첫번째노선", "BLUE", 1L, 2L, 10L));
```

Comment:
```suggestion
        Station 첫번째역 = stationRepository.save(new Station( "첫번째역"));
        Station 두번째역 = stationRepository.save(new Station( "두번째역"));
        Station 세번째역 = stationRepository.save(new Station( "세번째역"));
        LineResponse lineResponse = lineService.saveLine(new LineRequest("첫번째노선", "BLUE", 첫번째역.getId(), 두번째역.getId(), 10L));
```

>진행 도중에 유닛 테스트 때문에 production 코드를 access를 변경했습니다.
Mock 아닐때 해당 이슈가 생기긴 하는데 미래에 해당 문제가 생길때 보통 어떻게 대처하시는지 여쭤보고 싶습니다.
아마 제어할 수 있는 부분 vs 제어 할 수 없는 부분(access를 제한 해야하는부분) 으로 나눠질 것같은데 이야기를 들어보고싶습니다. 

질문하셨던 부분이 이런 부분인 것 같은데, 현재로서는 굳이 Station 의 id 를 넣는 생성자를 열어줄 필요까지는 없어보여요 😅 
강의중에 이야기를 했거나, 하실거 같은데, `생성자 정도는 테스트를 위해서 열어도 되지 않나?` 라는 이야기를 들었던 적은 있습니다.
개인적으로는 되도록이면 테스트를 위해 접근제어자 레벨을 변경하는 행위는 피하는 편이고, 생성자를 제외한 다른 경우에 접근제어자 레벨을 변경해야 할 것 같은 느낌이 오면, 클래스 분리가 덜 된 것이 아닌지 의심해봐야 한다고 생각합니다.

[private method test 에 관한 이야기](https://stackoverflow.com/questions/105007/should-i-test-private-methods-or-only-public-ones)